content/docs/user-guide/gems/reference/multiplayer/multiplayer-debug/debug/desync-audit-trail.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Multiplayer Desync Audit Trail
+description: Analyze and Debug Multiplayer Desyncs
+linkTitle: Desync Audit Trail
+---
+
+When the Client and Server disagree on the value of a networked variable, a desync occurs. Desyncs can be extremely difficult to debug which is where the Multiplayer Desync Audit Trail can help.
```

Comment:
This is the tooling you'd use after noticing desync log messages right? Might be worth mentioning _how_ someone can find out the simulation is desyncing.